src/main/scala/cromwell/backend/BackendValidationActor.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendValidationActor.{Validate, ValidationResult}
+import wdl4s.NamespaceWithWorkflow
+
+import scala.concurrent.Future
+
+object BackendValidationActor {
```

Comment:
I don't think so. We still have a functional requirement that a submitted workflow can be validated at submission time. What we discussed in person is what that means is that cromwell *can* run it, so as long as a single loaded backend would accept it then it would be fine. 

The idea as I understand it is that when the validation happens it'll also request backend specific validation against all loaded backends and ensure that at least one will accept it